build/config/RepoUtilData.json

```diff
@@ -34,6 +34,7 @@
                 "Microsoft.CodeAnalysis.Analyzers",
                 "Microsoft.VisualBasic",
                 "Microsoft.Composition",
+                "MicroBuild.*",
```

Comment:
Latest commit on this PR updated the README.md for RepoUtil to describe this section.